src/test/java/LoginTests.java

```diff
@@ -171,19 +160,16 @@ public void SearchSong () throws InterruptedException {
         passwordField.click();
         passwordField.sendKeys("te$t$tudent");
         submitButton.click();
-        Assert.assertTrue(driver.findElement(avatarSelector).isDisplayed());
-        // WebElement songsMenu = driver.findElement(By.className("songs"));
-        // WebElement songsMenu = driver.findElement(songMenuSelector);
-        driver.findElement(searchBarSelector).click();
-        driver.findElement(searchBarSelector).sendKeys("Veggie Straws");
-        //Assert.assertTrue(driver.findElement().isDisplayed());
+        Assert.assertTrue(getDriver().findElement(avatarSelector).isDisplayed());
+        getDriver().findElement(searchBarSelector).click();
+        getDriver().findElement(searchBarSelector).sendKeys("Veggie Straws");
```

Comment:
Using POM structure allows you to write tests using page actions without having to worry about the logic behind them. Now that you have a LoginPage object, you would be be able to write Login Page tests quick and easy, for 

For example

public class LoginTests extends BaseTest {

 ```
 @Test
  public void LoginEmptyEmailPasswordTest() {
    String expectedUrl = "https://bbb.testpro.io/";

    Assert.assertEquals(getDriver().getCurrentUrl(), expectedUrl);
  }

  @Test
  public void LoginValidEmailPasswordTest() {
    LoginPage loginPage = new LoginPage(getDriver());
    HomePage homePage = new HomePage(getDriver());

    loginPage.login();

    Assert.assertTrue(homePage.isUserAvatarDisplayed());
  }
}
```
![image](https://user-images.githubusercontent.com/104761482/201465247-523bbf0c-a765-42d4-bc67-5ca73164df15.png)


The POM pattern uses page objects that represent a web page (or a form within a page, if applicable) to separate test code (validations and test flow logic, for example) from page specific code. It does so by making all actions that can be performed on a page available as methods of the page object representing that page.

For example:

- LoginPage = LoginTest
- HomePage = HomeTest
- CurrentQueuePage = CurrentQueueTest